lang/sv/index.md

```diff
@@ -0,0 +1,249 @@
+---
+layout: swedish
+title: Semantisk versionshantering 2.0.0
+---
+
+Semantisk versionshantering 2.0.0
+==============================
+
+Sammanfattning
+-------
+
+Givet ett versionsnummer MAJOR.MINOR.PATCH, räkna upp:
```

Comment:
Yep me too

Get Outlook for iOS




On Thu, Aug 4, 2016 at 5:49 PM +0100, "Pär Björklund" <notifications@github.com> wrote:





> @@ -0,0 +1,249 @@
> +---
> +layout: swedish
> +title: Semantisk versionshantering 2.0.0
> +---
> +
> +Semantisk versionshantering 2.0.0
> +==============================
> +
> +Sammanfattning
> +-------
> +
> +Givet ett versionsnummer MAJOR.MINOR.PATCH, räkna upp:

Looks good to me

---
You are receiving this because you were mentioned.
Reply to this email directly or view it on GitHub:
https://github.com/mojombo/semver.org/pull/142/files/e46852c8583e765cc420cb80610764c40329c00a#r73559531